flatdata-generator/flatdata/generator/templates/rust/structure.jinja2

```diff
@@ -14,5 +14,11 @@ define_struct!(
     {%- if not loop.last %},
     {% endif %}
     {% endfor %}
+    {% for field in struct.fields %}
```

Comment:
Can't this loop be merged with the previous one?

Reply:
No, the macro definition does not allow mixing different field types (heterogeneous lists), but requires on list type to be followed by the other. Do you know if heterogeneous lists are possible?

Reply:
I would leave this as they are for now, and think about simplifying macros in general in scope of #136